### DISCUSSION TITLE
Grafana & Elasticsearch: Timezone in Count (Time Series) is different the Timezone in Table

### DISCUSSION THREAD

**Discussion by carlosmacapuna** [grafana/grafana#63224] [Dashboards, Panels & Visualizations]
I can't generate a counter chart with the same timeline. This has confused users. Basically, the two identical queries return different timezones. The images are explanatory.

I've tried several solutions to solve this problem of mine and I didn't find it.

Problem:

![grafana_elastic_01](https://user-images.githubusercontent.com/27429906/217929464-c0e745a0-8fbf-43aa-97ad-560a656ab994.png)

![grafana_elastic_02](https://user-images.githubusercontent.com/27429906/217929496-2bb426f3-179e-4959-9710-e5991656c70d.png)

![grafana_elastic_03](https://user-images.githubusercontent.com/27429906/217929517-d108bda5-6385-477b-968b-cdf44211e565.png)

Queries: 
![grafana_elastic_04](https://user-images.githubusercontent.com/27429906/217932819-56587080-f698-429f-82c9-c9a4dc15ed0a.png)

![grafana_elastic_05](https://user-images.githubusercontent.com/27429906/217932875-f1ef9bab-f48b-412e-bce8-2ea985d9c931.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82856. Feel free to continue the discussion around this there. Thank you!
